airbyte-integrations/connectors/source-github/source_github/schemas/comments.json

```diff
@@ -11,6 +11,65 @@
     "node_id": {
       "type": ["null", "string"]
     },
+    "user": {
+      "type": ["null", "object"],
+      "properties": {
+        "login": {
```

Comment:
as a potential improvement you can use references here to avoid data duplication in schema  , but I do not insist on that

Reply:
For future development, it probably better to have the whole schema in a single json, for each stream rather than use the $ref and deal with this dependencies after all, but if it's needed - there is no problem to modify this. 
For now, let's keep it as it is. WDYT?

Reply:
I'm fine

Reply:
this subobject 'user' is wanting a separate schema file)

Reply:
Done)